File: pinyougou-parent/pinyougou-pay-interface/src/main/java/com/pinyougou/pay/service/PayService.java
package com.pinyougou.pay.service;

import java.util.Map;

public interface PayService {

    Map<String, String> createNative(String orderPayNo, String money) throws Exception;

    Map<String, String> queryPayStatus(String orderPayNo) throws Exception;

    Map<String, String> closePay(String orderPayNo) throws Exception;
}

File: pinyougou-parent/pinyougou-seckill-web/src/main/webapp/plugins/angularjs/service/payService.js
app.service('payService', function ($http) {

    this.createNative = function () {
        return $http.post('../pay/createNative.do');
    };

    this.queryPayStatus = function (orderPayNo) {
        return $http.get('../pay/queryPayStatus.do?orderPayNo=' + orderPayNo);
    };
});